spec/classes/plugin_ansible_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe 'pulpcore::plugin::ansible' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:facts) { os_facts }
+      let(:pre_condition) { 'include pulpcore' }
+      let(:expected_vhost_content) do
+<<CONTENT
+
+  ProxyPass /pulp_ansible/galaxy/ unix:///run/pulpcore-api.sock|http://pulpcore-api/pulp_ansible/galaxy/
+  ProxyPassReverse /pulp_ansible/galaxy/ unix:///run/pulpcore-api.sock|http://pulpcore-api/pulp_ansible/galaxy/
+CONTENT
+      end
+
+      it { is_expected.to compile.with_all_deps }
+      it { is_expected.to contain_package('python3-pulp-ansible') }
+      it { is_expected.to contain_pulpcore__plugin('ansible') }
+      it do
+        is_expected.to compile.with_all_deps
+        is_expected.to contain_pulpcore__plugin('ansible')
+          .that_subscribes_to('Class[Pulpcore::Install]')
+          .that_notifies(['Class[Pulpcore::Database]', 'Class[Pulpcore::Service]'])
+        is_expected.to contain_apache__vhost__fragment('pulpcore-https-plugin-ansible')
+                           .with_content(expected_vhost_content)
```

Comment:
```suggestion
          .with_content(
            Regexp.new <<~CONTENT
              \n\s\sProxyPass /pulp_ansible/galaxy/ unix:///run/pulpcore-api.sock|http://pulpcore-api/pulp_ansible/galaxy/
              \s\sProxyPassReverse /pulp_ansible/galaxy/ unix:///run/pulpcore-api.sock|http://pulpcore-api/pulp_ansible/galaxy/
            CONTENT
          )
```

This works for me, has more readable indentation, and avoids the `let` block entirely. What do you think @jlsherrill @ekohl ?

Thanks to you both for going down this rabbit hole with me. I learned a lot from this :)

Reply:
I think that the current form is the most readable. We use the same pattern in multiple places.

Reply:
It looks horrible in the narrow suggested change window due to line wrapping, but much nicer inside any editor

Reply:
I'm not as big of a fan due to the \s characters personally.  All of this is kinda personal preference i guess

Reply:
IMO the `let` block is unnecessary because it's only used once. I am an rspec noob though so please `let` (har) me know if I am missing something. :)

With that said, while the current implementation is not my favorite due to the indentation, I see no reason to delay merging this further over style matters. If you are satisfied, please go ahead